extensions/Khronos/KHR_materials_cmnConstant/README.md

```diff
@@ -0,0 +1,118 @@
+# KHR\_materials\_cmnConstant
+
+## Contributors
+
+* Don McCurdy, Google, <mailto:donmccurdy@google.com>
+
+## Status
+
+Draft (not ratified yet)
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines an unlit ("constant") shading model for use in glTF 2.0
+materials, as an alternative to the Physically Based Rendering (PBR) shading
+models provided by the core specification. In mobile devices with limited
+resources, unlit materials offer a performant alternative to higher-quality
+shading models.
+
+## Extending Materials
+
+The common Constant material is defined by adding the
+`KHR_materials_cmnConstant` extension to any glTF material. When present, the
+extension indicates that a material should use the Constant shading model.
+Unrelated default values in the default PBR shading model, such as
```

Comment:
Can we be explicit about exactly which values should be ignored or is that something that is up to implementation?

Reply:
Added explicit comment on `metallic` and `roughness`. 

I'm not sure what a client should do if it sees KHR_materials_cmnConstant _and_ KHR_materials_pbrSpecularGlossiness. Inclined to leave that up for application choice.

Reply:
What about `metallicRoughnessTexture`? What about vertex attributes like `NORMAL`, `TANGENT` and `normalTexture`? Should these be ignored?

Reply:
> KHR_materials_cmnConstant and KHR_materials_pbrSpecularGlossiness

Yeah, this one is a bit tricky. My gut feel is that this should not happen. If an asset has an unlit material, it probably wants to be fast, and I can't think of a good reason to have specular glossiness there also.